http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/MockRuntimeServiceInterp.scala

```diff
@@ -27,6 +27,15 @@ class BaseMockRuntimeServiceInterp extends RuntimeService[IO] {
 
   override def listRuntimes(userInfo: UserInfo, cloudContext: Option[CloudContext], params: Map[String, String])(
     implicit as: Ask[IO, AppContext]
+  ): IO[Vector[ListRuntimeResponse2]] =
+    listRuntimes(userInfo, cloudContext, params, List.empty)
```

Comment:
I'm not sure how much it matters, but the logic here in the mock is not consistent with the logic in the real `RuntimeServiceInterp`. In `RuntimeServiceInterp`, this uses:
```scala
    val excludeStatuses = List(RuntimeStatus.Deleted)
    listRuntimes(userInfo, cloudContext, params, excludeStatuses)
```